.github/workflows/test.yml

```diff
@@ -23,6 +23,12 @@ jobs:
       - name: Checkout the repo
         uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
 
+      # Workaround to provide additional free space for testing.
+      # Reference: https://github.com/actions/runner-images/issues/2840
+      - name: Free disk space
+        if: startsWith(runner.name, 'GitHub Actions') && startsWith(matrix.os, 'ubuntu')
+        run: sudo rm -rf /usr/share/dotnet
```

Comment:
Since yesterday...at least within this branch. There is a larger discussion about this in Slack. Bottom line is that the runner images are only guaranteed to have _at least_ 14GB of disk space available, they often have more, but can dip down to 14GB _at any time_.